intern/service/dvid/volume.py

```
@@ -127,7 +127,8 @@ def get_cutout(self, resource, resolution, x_range, y_range, z_range, **kwargs):
                 resource.name, resp.status_code, resp.text))
             raise HTTPError(msg, response=resp)
         
-        cutout = blosc.unpack_array(resp.content)
+        block = np.fromstring(resp.content, dtype = kwargs.get("dtype", np.uint8))
```

Do we want to force them to provide the data type instead of silently defaulting to uint8 if none provided?

---

I like to think it is safe to assume a uint8 data type since it looks like that is typically what you would find in their data instances. Failure to specify the right data type does result in an unclear error though, so I could clarify the error, or force a data type....

---

I should also be using the datatype as defined by the resource and not an independently defined one. So I'll fix this